scripting/include/srccoop/manager.inc

```diff
@@ -66,33 +66,46 @@ methodmap ChangeLevelManager < Array_t
 	public void UpdateStatus(int iSecondsLeft)
 	{
 		char szMapName[MAX_MAPNAME], szDisplay[128];
-		Panel panel = new Panel();
-		
-		Format(szDisplay, sizeof(szDisplay), "Changing map.. (%d)", iSecondsLeft);
-		panel.SetTitle(szDisplay);
-
-		ChangeLevelManager pSorted = this.Sorted();
-		ChangeLevelVoteLump pChangeLvlLump;
-		
-		for (int i = 0; i < pSorted.Length; i++)
+		if (g_pConvarEndWaitDisplayMode.BoolValue)
 		{
-			pSorted.GetArray(i, pChangeLvlLump);
-			if(pChangeLvlLump.m_pChangeLevel.IsValid())
+			Format(szDisplay, sizeof(szDisplay), "Changing map\n(%d)", iSecondsLeft);
```

Comment:
I think for multiple maps it would be better to leave it as a panel, I will just put a check for if `!pSorted.Length` and use the panel in those cases.
The HudText can do multiple lines, but with even 3 or 4 extra lines it can take up a chunk of your screen from the text size.

Reply:
I decided to enable it as 
![image](https://user-images.githubusercontent.com/31295298/148631415-3be62c89-aed9-4910-8957-bca56b141508.png)
The additional map names will appear next to it so it would look like: 
`bm_c4a1c1 (1 players) bm_c4a1c2 (# players)`
This is center justified, the issue then would only occur if there ever were to be too many maps in the queue, it would scroll off on the left and right edges of the screen, but that would take roughly 7+ (at the regular bm_c#a#a character count) in the queue.

Reply:
I could also put in a CVar or some configurations for the position and colors of this message.